packages/desktop-ui-lib/src/lib/settings/plugins/component/plugin-marketplace/plugin-marketplace-detail/plugin-marketplace-detail.component.html

```
@@ -275,6 +275,20 @@
 							}
 						</span>
 					</nb-toggle>
+					<!-- Web-to-Desktop Installation Button -->
+					@if(!isDesktopApp() && !installed) {
+					<button
+						nbButton
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)